sdk/confidentialledger/Azure.Security.CodeTransparency/README.md

```diff
@@ -45,18 +45,32 @@ To submit the signature, use the following code:
 CodeTransparencyClient client = new(new Uri("https://<< service name >>.confidential-ledger.azure.com"), null);
 FileStream fileStream = File.OpenRead("signature.cose");
 BinaryData content = BinaryData.FromStream(fileStream);
-Operation<GetOperationResult> operation = await client.CreateEntryAsync(content);
-Response<GetOperationResult> operationResult = await operation.WaitForCompletionAsync();
-Console.WriteLine($"The entry id to use to get the entry and receipt is {{{operationResult.Value.EntryId}}}");
-Response<BinaryData> signatureWithReceiptResponse = await client.GetEntryAsync(operationResult.Value.EntryId, true);
+Operation<BinaryData> operation = await client.CreateEntryAsync(content);
+Response<BinaryData> operationResult = await operation.WaitForCompletionAsync();
+
+string entryId = string.Empty;
+CborReader cborReader = new CborReader(operationResult.Value);
+cborReader.ReadStartMap();
+while (cborReader.PeekState() != CborReaderState.EndMap)
+{
+    string key = cborReader.ReadTextString();
+    if (key == "EntryId")
+    {
+        entryId = cborReader.ReadTextString();
+    }
+    else
+        cborReader.SkipValue();
+}
+Console.WriteLine($"The entry id to use to get the entry and receipt is {{{entryId}}}");
+Response<BinaryData> signatureWithReceiptResponse = await client.GetEntryAsync(entryId);
 BinaryData signatureWithReceipt = signatureWithReceiptResponse.Value;
 byte[] signatureWithReceiptBytes = signatureWithReceipt.ToArray();
 ```
 
 Once you have the receipt and the signature, you can verify whether the signature was actually included in the Code Transparency service by running the receipt verification logic. The verifier checks if the receipt was issued for a given signature and if the receipt signature was endorsed by the service.
 
 ```C# Snippet:CodeTransparencyVerification
-CcfReceiptVerifier.RunVerification(signatureWithReceiptBytes);
+client.VerifyTransparentStatementReceipt(signatureWithReceiptBytes, content.ToArray());
```

Comment:
When verifying `signatureWithReceiptBytes` second argument should be `null` I think

Reply:
The verification flow is taking the input signed statement bytes to validate it against the claim digest.
Are you suggesting adding a new verification method that doesn't include the claims digest validation?

Reply:
Updated the logic using the latter approach. We now rely only on the transparentStatement